### PR TITLE
Added install step for devs new to js/npm

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,11 @@
 
 A function that monkey patches React and notifies you in the console when **potentially** unnecessary re-renders occur. Super helpful for easy perf gainzzzzz.
 
+### Install
+```js
+npm install --save why-did-you-update
```
Comment (leandrooriente) [inline README.md]: --save-dev 😉

**Comment by leandrooriente** [comment]
--save-dev 😉

**Comment by amilajack** [comment]
@leandrooriente sorry about that! Fixed :)

**Comment by moklick** [comment]
\`\`\`shell :wink:

**Comment by amilajack** [comment]
@garbles what is the status of this?

**Comment by garbles** [comment]
Thanks!

**Pull request merged**